bolt/test/AArch64/simple-negate-ra-state.s

```diff
@@ -0,0 +1,42 @@
+# RUN: llvm-mc -filetype=obj -triple aarch64-unknown-unknown %s -o %t.o
+# RUN: %clang %cflags  %t.o -o %t.exe -Wl,-q
+
+# RUN: llvm-objdump %t.exe -d > %t.exe.dump
+# RUN: llvm-objdump --dwarf=frames %t.exe -D > %t.exe.dump-dwarf
```

Comment:
Do we need `-disassemble-all` here, or would `-d` be enough?
Or better yet, since python processes a single function, can use:
`--disassemble-symbols=foo`

Note: I noticed your bolted `.dump-dwarf` doesn't include any `.text`.
Still, you could use a single file that contains just the `.text` of foo.
What do you think?

Reply:
I think it is better to have the dwarf info and the normal disassembly in separate files, but disassembling one foo is definately better than using the `-d` flag.